client/component/ToggleGroup.tsx

```diff
@@ -0,0 +1,101 @@
+import { useContext, createContext } from 'react';
+import type { ReactElement, ComponentType, SVGAttributes } from 'react';
+
+import styles from './ToggleGroup.module.css';
+
+interface ToggleGroupProps {
+  role: 'tablist' | 'radiogroup';
+  value: string;
+  disabled?: boolean;
+  onChange: (val: string) => void;
+}
+
+const ToggleGroupContext = createContext<ToggleGroupProps | undefined>(
+  undefined
+);
+
+function useToggleGroupProps(): ToggleGroupProps {
+  const context = useContext(ToggleGroupContext);
+
+  if (!context) {
+    throw new Error('Missing Toggle Group provider.');
+  }
+
+  return context;
+}
+
+interface BtnProps {
+  label: string;
+  value: string;
+  icon?: ComponentType<SVGAttributes<SVGElement>>;
+  iconOnly?: boolean;
+  disabled?: boolean;
+  position?: 'left' | 'middle' | 'right';
+}
+
+function Btn(props: BtnProps) {
+  const {
+    label,
+    value,
+    icon: Icon,
+    iconOnly,
+    disabled = false,
+    position = 'left',
+  } = props;
+  const {
+    role,
+    value: selectedValue,
+    disabled: isGroupDisabled,
+    onChange,
+  } = useToggleGroupProps();
+
+  let btn_class: string;
```

Comment:
I think you could make your own button with the tooltip inside it and pass the text as a prop then you re-use `h5web/lib`'s `ToggleGroup`